mobile-app/app/components/icons/assets/index.test.tsx

```diff
@@ -9,7 +9,7 @@ import { Provider } from 'react-redux'
 jest.mock('randomcolor', () => jest.fn().mockReturnValue('#ffffff'))
 
 const icons = ['_UTXO', 'DFI', 'DFI (UTXO)', 'DFI (Token)', 'BTC', 'dBCH', 'dBTC', 'dDFI', 'dDOGE', 'dETH', 'dLTC',
-  'dUSDT', 'dUSDC', 'FAKE', 'dDUSD', 'DUSD', 'dTSLA']
+  'dUSDT', 'dUSDC', 'FAKE', 'dDUSD', 'DUSD', 'dTSLA', 'dGOOGL']
```

Comment:
@fullstackninja864 can you add all possible tokens here?

Reply:
Sure I will add all possible token available